torch/csrc/jit/mobile/import.cpp

```diff
@@ -301,12 +301,25 @@ void BytecodeDeserializer::parseMethods(
       OpCode op_code = parseOpCode(ins_item[0].toString()->string().c_str());
       int X = ins_item[1].toInt();
       int N = ins_item[2].toInt();
-      function->append_instruction(op_code, X, N);
+      // TODO: Save debug handles for all instructions, not just for OP
       if (op_code == OP) {
-        std::string module_debug_info = (has_debug_info)
-            ? module_debug_info_list[X].toString()->string()
-            : "";
-        function->set_module_info(module_debug_info, i);
+        // In later PRs we will refactor this to always save debug handles.
+        // Debug info, source range and inlined callstack ptr saving will become
+        // optional.
+        if (has_debug_info) {
+          auto module_debug_tuple =
+              module_debug_info_list[X].toTuple()->elements();
+          std::string module_debug_info =
+              module_debug_tuple[0].toString()->string();
+          int64_t debug_handle = module_debug_tuple[1].toInt();
+          function->set_module_info(module_debug_info, i);
```

Comment:
This will change in subsequent diffs. I kept it here for incremental changes. Later on the only thing we will have is debug handle. And then how the debug handles get symbolicated is build time thing.